pom.xml

```diff
@@ -6,7 +6,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.sonian</groupId>
   <artifactId>elasticsearch-jetty</artifactId>
-  <version>1.7.3-BV</version>
+  <version>1.7.3-SNAPSHOT</version>
```

Comment:
Is this what we want, pre-release? (Is there a jenkins job that does the release to bump the actual released version to `1.7.3`?)